syft/execution/computation.py

```diff
@@ -130,15 +134,17 @@ def bufferize(worker: AbstractWorker, action: "ComputationAction") -> "Computati
                 )
 
         if action.return_ids is not None:
-            if type(action.return_ids) == PlaceHolder:
-                return_ids = list((action.return_ids,))
+            if not isinstance(action.return_ids, (list, tuple)):
+                return_ids = (action.return_ids,)
             else:
                 return_ids = action.return_ids
 
             for return_id in return_ids:
-                if type(return_id) == PlaceHolder:
-                    protobuf_op.return_placeholders.extend(
-                        [sy.serde.protobuf.serde._bufferize(worker, return_id)]
+                if isinstance(return_id, PlaceholderId):
+                    # NOTE to know when we have a PlaceholderId, we store it
+                    # in return_placeholder_ids and not in return_ids
```

Comment:
Is this still true? Line 197 below looks like it's using the `return_ids` field.

Reply:
Yep, but in the protobuf schema, I use the `return_placeholder_ids` field for placeholderId and the `return_ids` field for classic ids.
Maybe I don't need to do so?